Modules/Core/Mesh/src/itkTriangleMeshCurvatureCalculator.cxx

```diff
@@ -0,0 +1,42 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#include "../include/itkTriangleMeshCurvatureCalculator.h"
```

Comment:
ITK never uses relative imports. `#include "itkTriangleMeshCurvatureCalculator.h"` should be enough. If not, then the issue is more likely to come from something missing in the relevant `CMakeLists.txt` files I believe.

Reply:
ok, I saw this in another file so used the same way. I will change it.

Reply:
So should we test the enum ostream  ?
Is there a relevant macro for it ? Or just string comparison is needed ?

Reply:
String comparison would be best, but no other enum class ostream overload has done it so far, so you may want to tackle that in a separate PR across all the code base if you have the time. Or open an issue.

So far we have contented ourselves printing the values, e.g.
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/IO/NIFTI/test/itkNiftiImageIOTestHelper.cxx#L132

Reply:
Visual Studio sometimes changes include statements to be relative like that. If you saw it somewhere else in ITK, it should be fixed too.

Reply:
There are many files I found by grepping. Creating a separate PR for removing relative import.